### PR TITLE
[GCMSLG-83] Fix blog content type tests.

### PR DESCRIPTION
**Pull request by Ahmedjabar**
Fixed Blog content type behat test.

### CODE REVIEW

File: tests/behat/features/content_types/blog-article.feature [hunk 1]
```diff
@@ -32,6 +28,10 @@ Feature: Blog Article
     And I submit the media browser
     Then the "#edit-field-thumbnail" element should contain "Edit"
     And the "#edit-field-thumbnail" element should contain "Remove"
+    And I fill in the following:
+      | Title   | New blog               |
+      | Summary | How we migrated to govCMS! |
+    And I put "Digital transformation is real. GovCMS is the best!" into WYSIWYG of "Body" field
     Given I click "Publishing options"
```
Comment (gargsuchi) [inline tests/behat/features/content_types/blog-article.feature:35]: @Ahmedjabar  Why do we need to change the order of data entry here?

File: tests/behat/features/content_types/blog-article.feature [hunk 2]
```diff
@@ -64,10 +64,11 @@ Feature: Blog Article
   @api @javascript
   Scenario: Check that moderation works.
     Given "blog_article" content:
-      | title       | author     | status | state |
-      | Agency blog | Jim Editor | 0      | draft |
+      | title       | author     | status | workbench_moderation_state_new |
+      | Agency blog | Jim Editor | 0 | draft |
     And I am logged in as a user with the "Content approver" role
     When I am on "/news-media/blog/agency-blog"
+    And save screenshot
```
Comment (gargsuchi) [inline tests/behat/features/content_types/blog-article.feature]: @Ahmedjabar Was this added as a debug? Do we need it in actual tests?

**Review by gargsuchi — CHANGES_REQUESTED** [review]
Please check my questions.

File: tests/behat/features/content_types/blog-article.feature (hunk 1 above)
Comment (Ahmedjabar) [reply tests/behat/features/content_types/blog-article.feature:35]: @gargsuchi the page scrolls down when the body is being filled which places the tag and featured image fields behind the admin bar, thus they become un-clickable.

File: tests/behat/features/content_types/blog-article.feature (hunk 2 above)
Comment (Ahmedjabar) [reply tests/behat/features/content_types/blog-article.feature]: Good catch! yes it was for debugging only. I will remove.

**Pull request merged**